### PR TITLE
added dark mode in about us page and services page

### PR DESCRIPTION
**Pull request by Deepika14145**
# 🚀 Pull Request

## Description

closes https://github.com/07sumit1002/CabRental/issues/211

Please include a summary of the changes and the related issue. Provide relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of Change

- [+] New feature (non-breaking change which adds functionality)
- added dark mode in about us page and services page

## Checklist:

- [+] My code follows the style guidelines of this project
- [+] I have performed a self-review of my own code
- [+] I have commented my code, particularly in hard-to-understand areas
- [+] I have made corresponding changes to the documentation
- [+] My changes generate no new warnings
- [+] I have added tests that prove my fix is effective or that my feature works
- [+] New and existing unit tests pass locally with my changes
- [+] Any dependent changes have been merged and published in downstream modules

## Screenshot of final output/video
about us page -
Screenshot 2024-10-06 163103
services page -
Screenshot 2024-10-06 163218

### CODE REVIEW

**Comment by Deepika14145** [comment]
Please review @sanjay-kv

File: about.html [hunk 1]
```diff
@@ -201,46 +228,59 @@ <h2>Company Overview</h2>
         <h2>Mission and Vision</h2>
         <div class="section-img">
           <img
-            src="https://imgs.search.brave.com/2CAFCbM6U5I6Q03dK-bC8qqqyb_mjrDxo4YGu4-zCnk/rs:fit:500:0:0/g:ce/aHR0cHM6Ly9pbWFn/ZXMuZnJlZWltYWdl/cy5jb20vaW1hZ2Vz/L2xhcmdlLXByZXZp/ZXdzLzMxOC9pbi10/aGUtY2FyLTEtMTQ5/NTA3MC5qcGc_Zm10"
-            alt="Mission and Vision Image"
+            src="https://imgs.search.brave.com/q0e8cW8B-XzUbrgZx4qP0ObqkV3g5cPE82Dptj32vVY/rs:fit:860:0:0/g:ce/aHR0cHM6Ly9pbWcu/ZnJlZXBpay5jb20v/ZnJlZS1waG90by9w/aG90by1jb250ZW50/LzIwMTkvMDMvbXNo/aXN0LzE1NTM5MDA2/bmF0dXJlcmVpbC9z/bGFkZXJzL3ZhbC9j/YWxlbmRhci5qcGc"
```
Comment (akbatra567) [inline about.html:231]: Could you download the image and import from local url ?
This image is not showing up. 

Screenshot 2024-10-07 at 00 39 26

File: about.html [hunk 2]
```diff
@@ -201,46 +228,59 @@ <h2>Company Overview</h2>
...
-          <li>
-            <strong>Innovation:</strong> We continuously seek innovative
-            solutions to enhance our services.
-          </li>
-          <li>
-            <strong>Sustainability:</strong> We are committed to sustainable
-            practices and reducing our environmental impact.
-          </li>
+          <li>Customer Satisfaction</li>
+          <li>Integrity</li>
```
Comment (akbatra567) [inline about.html:247]: Could you share your thoughts behind changing the content ? 

Also as the description is removed 

After:

File: about.html [hunk 3]
```diff
@@ -201,46 +228,59 @@ <h2>Company Overview</h2>
...
-          </li>
-          <li>
-            <strong>Innovation:</strong> We continuously seek innovative
-            solutions to enhance our services.
-          </li>
-          <li>
-            <strong>Sustainability:</strong> We are committed to sustainable
-            practices and reducing our environmental impact.
-          </li>
+          <li>Customer Satisfaction</li>
```
Comment (akbatra567) [inline about.html:246]: Could you share the thoughts behind changing the content ? 

Also after changing content there's a lot of white space that it feels empty. 

Suggestions: 
1. Experiment with text alignment.
2. Experiment with the box sizing if we even need that big values and principles boxes. 

Screenshot 2024-10-07 at 00 43 54

**Review by akbatra567 — CHANGES_REQUESTED** [review]
I see some common styles related to dark mode theme which is added to both `about.html` and `serv.html` and would be similar for other pages. Could you create an external css file and transfer the common css there and import that and verify if it works the same.

**Comment by Deepika14145** [comment]
I will check all above mentioned issues and will work on ìt.

**Comment by Deepika14145** [comment]
Sorry for late actually my exams are going on I will update it soon @akbatra567

**Comment by Deepika14145** [comment]
Can you please review the changes that I have made @akbatra567 ? I have committed some changes.

**Comment by akbatra567** [comment]
> Hello, @akbatra567 Please review the changes I have made in PR.

Requested changes are missing: 

> I see some common styles related to dark mode theme which is added to both about.html and serv.html and would be similar for other pages. Could you create an external css file and transfer the common css there and import that and verify if it works the same.


Let me explain this a bit on what you need to do: 
1. See what all common styles you are using for the 2 html pages.
2. create a common css file name could be anything you feel adds more value, suggestions: `dark.css` - in case you only copy dark theme styles, else could be `main.css` / `common.css`
3. Import this newly created css file inside the html files replacing the duplicated code from the files. using `<link rel="stylesheet" href='<add file path here>'>`
4. Verify the styles if it working as expected. 
5. Tag me when you complete these.

**Comment by Deepika14145** [comment]
ok @akbatra567 i will update it soon

**Comment by Deepika14145** [comment]
If this PR is merged then why its not showing "Accepted" on hacktoberbest website @07sumit1002

**Comment by Deepika14145** [comment]
please add hacktoberfest-accepted labels to this PR @07sumit1002 and labels

**Pull request merged**